dokka-integration-tests/gradle/src/testTemplateProjectAndroid/kotlin/Android0GradleIntegrationTest.kt

```diff
@@ -43,7 +42,7 @@ class Android0GradleIntegrationTest : AbstractGradleIntegrationTest() {
     @ArgumentsSource(AndroidTestedVersionsArgumentsProvider::class)
     fun execute(buildVersions: BuildVersions) {
         val result = createGradleRunner(buildVersions, "dokkaHtml", "-i", "-s").buildRelaxed()
-        assertEquals(TaskOutcome.SUCCESS, assertNotNull(result.task(":dokkaHtml")).outcome)
+        result.shouldHaveTask(":dokkaHtml").shouldHaveOutcome(SUCCESS, FROM_CACHE)
```

Comment:
just curious: in which case it will be `FROM_CACHE`?

Reply:
One way would is:

1. Run the tests locally (thus populating the TestKit cache in `dokka-integration-tests/gradle/build/gradle-test-kit`),
2. then you edited `dokka-integration-tests/gradle/build.gradle.kts`.
3. The tests would re-run, but because the test project hasn't changed, the tasks could be loaded from cache.